Integrations/SentinelOne-V2/SentinelOne-V2.yml

```diff
@@ -1259,6 +1261,209 @@ script:
     description: Broadcasts a message to all agents that match the input filters.
     execution: false
     name: sentinelone-broadcast-message
+  - arguments:
+    - default: false
+      description: 'Limit number of returned items (1-100). Example: "10".'
+      isArray: false
+      name: limit
+      required: false
+      secret: false
+    - default: false
+      description: 'QueryId obtained when creating a query under Create Query. Example:
+        "q1xx2xx3".'
+      isArray: false
+      name: query_id
+      required: true
+      secret: false
+    deprecated: false
+    description: Get all Deep Visibility events from query
+    execution: false
+    name: sentinelone-get-events
+    outputs:
+    - contextPath: SentinelOne.Event.ProcessUID
```

Comment:
might be confusing to return both uid and id, do we need both ?

Reply:
Asked the research team about that. They told me to return it.